comment/api/serializers.py

```diff
@@ -117,21 +117,15 @@ def create(self, validated_data):
         email = validated_data.get('email')
         time_posted = timezone.now()
 
-        comment = Comment(
+        temp_comment = Comment(
             content_object=self.context['model_obj'],
             content=content,
             user=user,
             parent=self.context['parent_comment'],
             email=email,
             posted=time_posted
         )
-        self.email_service = DABEmailService(comment, request)
-        if settings.COMMENT_ALLOW_ANONYMOUS and not user:
-            self.email_service.send_confirmation_request(api=True)
-        else:
-            comment.save()
-            if settings.COMMENT_ALLOW_SUBSCRIPTION:
-                self.email_service.send_notification_to_followers()
+        comment = self.perform_create(temp_comment, request, api=True)
         return comment
```

Comment:
I just like it in this form because it tells me explicitly what the function is returning rather than go(while debugging) and see inside the function to see.

Reply:
The variable is redundant here. you can add this line while debugging if this the case.